tests/conditional_compilation/conftest.py

```diff
@@ -75,10 +89,12 @@ def pytest_generate_tests(metafunc):
         for models in test:
             extra_args = {}
             model_path = models["model"]["path"]
+            model = models["model"]
```

Comment:
it can be a list of models (according to implementation), so further conversion to `Path` will lead to crash

Reply:
According to test config design, `models` is a dict with model info (variable name mislead me). Please, rename `models` to `model`